src/scripts/main.js

```diff
@@ -1,3 +1,340 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tableBody = table.querySelector('tbody');
+
+headers.forEach((header) => {
+  header.dataset.sorted = 'DESC';
+});
+
+function sortColumn(index, isASC) {
+  let sortedRows = [];
+
+  if (isASC) {
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
+    index === 4
+      ? sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(a.cells[index].innerHTML)
+        - salaryToNumber(b.cells[index].innerHTML)))
+      : sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (a.cells[index].innerHTML.localeCompare(b.cells[index].innerHTML)));
+  } else {
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
+    index === 4
+      ? sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(b.cells[index].innerHTML)
+        - salaryToNumber(a.cells[index].innerHTML)))
+      : sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (b.cells[index].innerHTML.localeCompare(a.cells[index].innerHTML)));
+  }
+
+  table.tBodies[0].append(...sortedRows);
+}
+
+headers.forEach((header, index) => {
+  header.addEventListener('click', (e) => {
+    let order;
+
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
+    e.target.dataset.sorted === 'ASC'
+      ? (e.target.dataset.sorted = 'DESC',
+      order = false)
+      : (e.target.dataset.sorted = 'ASC',
+      order = true);
```

Comment:
fix this 
<img width="1662" alt="image" src="https://user-images.githubusercontent.com/99403198/209543505-ac4cc3a7-d8a6-4124-96fb-3a319e2c9362.png">